src/containers/__init__.py

```diff
@@ -0,0 +1,31 @@
+from dataclasses import dataclass, asdict
+from typing import List, Optional, Union
+
+
+@dataclass
```

Comment:
I would move these classes to stand-alone files.
I think that __init__.py is not the correct place for these classes.

Reply:
Make this file empty, please. There are still old imports.